File: app/src/main/java/br/fib/tentaasorte/Cliente.java
package br.fib.tentaasorte;

import androidx.annotation.NonNull;

public class Cliente {

    private String nome;
    private String time;

    public String getTime(){
        return this.time;
    }

    public void setTime(String time) {
        this.time = time;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public Cliente(String nome, String time) {

        this.nome = nome;
        this.time = time;
    }

    @Override
    public String toString() {
        return this.nome;
    }
}

File: app/src/main/java/br/fib/tentaasorte/Cliente.kt
package br.fib.tentaasorte

class Cliente {

    private var nome: String = ""
    private var time: String = ""


    constructor(nome: String, time: String) {
        this.nome = nome
        this.time = time
    }

    override fun toString(): String {
        return nome
    }

    fun getTime() : String {
        return this.time
    }

    fun getNome(): String {
        return this.nome
    }

}